### PR TITLE
Update Polish (pl) localization of Mozilla Advocacy

### PR DESCRIPTION
**Pull request by piotrdrag**
~~needs-review: @splewako~~

### CODE REVIEW

File: locales/pl/copyright-email1.properties [hunk 1]
```diff
@@ -1 +1,23 @@
 copyright_email1_itstime=Czas zaktualizować prawo autorskie do 21 wieku. Podpisz petycję już dzisiaj.
+photographing_the_eiffel_tower=Zrobienie zdjęcia świateł Wieży Eiffla w nocy, remiks piosenki, wrzucenie mema. Może Cię zaskoczyć, że te trzy rzeczy mogą być nielegalne.
```
Comment (splewako) [inline locales/pl/copyright-email1.properties]: Please don't use "Cię". Maybe "Może Ciebie zaskoczyć, że te" or even better "Może zaskakiwać ale te …"?

File: locales/pl/copyright-email1.properties [hunk 2]
```diff
@@ -1 +1,23 @@
...
+Big_publishers=Wielkie wydawnictwa, dysponenci własnością intelektualną i inne firmy lobbują w instytucjach Unii Europejskiej na rzecz utrzymania przestarzałego prawa. Niektóre próbują uchwalić jeszcze ostrzejsze regulacje, zmuszające wyszukiwarki do płacenia za linkowanie do informacji prasowych w Internecie. Inne chcą uchwalenia nowego prawa upoważniającego do monitorowania i filtrowania sieci.
+We_need_law_changes=Potrzebujemy zmian w prawie, aby promować kreatywność i chronić przyszłe formy wyrażania się, o których jeszcze nawet nie słyszeliśmy. W przeciwnym razie możemy stracić najciekawsze i najdziwniejsze rzeczy 21 wieku. Dla mnie oznaczałoby to stratę przeuroczych memów (psy na deskorolkach!), które magicznie poprawiają mi humor.
+Together_we_can=Razem możemy ochronić kreatywność w Internecie – w 21 wieku i w jeszcze dalszej przyszłości. Podpiszesz petycję?
+Thanks_for_all_that_you=Dziękujemy za wszystko,
+Sara=Sara i zespół Mozilli
+Sara_Haghdoosti=Sara Haghdoosti, kierowniczka ds. kampanii i sieci, dział kwestii społecznych
+P_S=PS Jeśli chcesz otrzymywać wiadomości od nas w innym języku, napisz do nas.
```
Comment (splewako) [inline locales/pl/copyright-email1.properties]: "S." (from Sara) could be added at the end.

File: locales/pl/copyright-email1.properties [hunk 3]
```diff
@@ -1 +1,23 @@
...
+Image_Attribution=Powyższy obraz został utworzony z materiałów autorstwa Rubena de Rijcke (własna praca) za pośrednictwem serwisu Wikimedia Commons, Petra Kratochvila za pośrednictwem serwisu PublicDomainPictures.net, użytkownika Groink (własna praca) za pośrednictwem serwisu Wikimedia Commons oraz użytkownika smial (własna praca) za pośrednictwem serwisu Wikimedia Commons.
+Thanks_for_reading=Dziękujemy za uwagę!
+Youre_Receiving=Otrzymujesz tą wiadomość, ponieważ subskrybujesz biuletyn Mozilli. Jeśli nie chcesz ich otrzymywać, odsubskrybuj lub zmień preferencje.
```
Comment (splewako) [inline locales/pl/copyright-email1.properties]: tę wiadomość

File: locales/pl/copyright-email1.properties [hunk 4]
```diff
@@ -1 +1,23 @@
...
+Mozilla_Address=331 E. Evelyn Avenue Mountain View CA 94041
+Read_the_PP=Polityka prywatności Mozilli.
+We_have_received_your_request=Otrzymaliśmy Twoje żądanie
+Thank_you_for_submitting_your_request=Dziękujemy za wysłanie żądania. Naszym celem jest odpowiedź w 48 godzin, ale może to zająć trochę dłużej z powodu większej korespondencji niż zwykle.
+Your_support_means_so_much=Twoje wsparcie wiele dla nas znaczy. Odpowiemy jak najszybciej to możliwe.
```
Comment (splewako) [inline locales/pl/copyright-email1.properties]: Doesn't "Odpowiemy najszybciej jak to możliwe" sound better?

File: locales/pl/copyright-email1.properties (hunk 1 above)
Comment (piotrdrag) [inline locales/pl/copyright-email1.properties]: Changed to "Może to zaskakiwać, ale te..."

File: locales/pl/copyright-email1.properties (hunk 2 above)
Comment (piotrdrag) [inline locales/pl/copyright-email1.properties]: Added.

File: locales/pl/copyright-email1.properties (hunk 3 above)
Comment (piotrdrag) [inline locales/pl/copyright-email1.properties]: Changed. And I couldn't find it either.

File: locales/pl/copyright-email1.properties (hunk 4 above)
Comment (piotrdrag) [inline locales/pl/copyright-email1.properties]: It surely does.

**Comment by piotrdrag** [comment]
@ScottDowne please merge.

**Pull request merged**